Feature Guides/Features.md

```diff
@@ -1,13 +1,21 @@
-## Banners Toggling
+## SignOut
 
-You have an option to toggle, whether you want to show banners on the `Inbox Screen` or not. You just have to set a `BOOL` on the `Haptik.h` class named `showBanners`.
-
-Example:
+To Signout the current user from Haptik, use the following method on `[Haptik sharedSDK]`:
 
 ```
-[Haptik sharedSDK].showBanners = YES;
+[[Haptik sharedSDK] signoutFromHaptik:^(BOOL success, NSError * _Nullable error) {
+
+if (success) {
+
+}
+else {
+
+}
+}];
 ```
 
+> **Note**: You have to set the device token again after the SignOut.
```

Comment:
Does the developer needs to re-initialize HaptikLib again for using it?
Scenario where the user signs out and then visit the Haptik feature again without quitting the app. @iamsimranjot

Reply:
No @tejas-haptik. HaptikLib does not require any initialisation. It's a singleton class.

Reply:
If you are not clearing caches, references, application reference etc. then it's perfect 👍 
And MQTT etc get's re-connected if new user signs in after log out @iamsimranjot